### ISSUE TITLE
`succeed`, `precede` and `surround` HTML escaping

### ISSUE THREAD

**Issue by felixbuenemann** [k0kubun/hamlit#35]
The helpers  `succeed`, `precede` and `surround` should not escape html.

Example:
```haml
= surround '(', ')' do
  %i foo
```

Should output:
```html
(<i>foo</i>)
```

Actual output:
```html
(&lt;p&gt;foo&lt;/p&gt;)
```

**Comment by k0kubun** [comment]
Fixed in 0d2e3a9407cb9d3cf6a3d5ee28ca8a827745bf91 and released as v1.7.1. :gem: 
Thank you for reporting an issue.

**Comment by felixbuenemann** [comment]
Thank you for the quick fix! :heart:
